cvat/apps/engine/views.py

```diff
@@ -1317,6 +1339,24 @@ def dataset_export(self, request, pk):
             return Response(data="Exporting a dataset from a task without data is not allowed",
                 status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(summary='Method returns a preview image for the task',
+        responses={
+            '200': OpenApiResponse(description='Task image preview'),
```

Comment:
@azhavoro , what it will return if a task does not have any images?

Reply:
fixed